cl/corpus_importer/management/commands/financial_disclosures.py

```diff
@@ -0,0 +1,344 @@
+        disclosure.save()
+
+        for investment in extracted_data["sections"]["Investments and Trusts"][
+            "rows"
+        ]:
+            Investment.objects.create(
+                financial_disclosure=disclosure,
+                redacted=True
+                in [v["is_redacted"] for _, v in investment.items()],
+                description=investment["A"]["text"],
+                has_inferred_values=investment["A"]["inferred_value"],
+                income_during_reporting_period_code=investment["B1"]["text"],
+                income_during_reporting_period_type=investment["B2"]["text"],
+                gross_value_code=investment["C1"]["text"],
+                gross_value_method=investment["C2"]["text"],
+                transaction_during_reporting_period=investment["D1"]["text"],
+                transaction_date_raw=investment["D2"]["text"],
+                transaction_value_code=investment["D3"]["text"],
+                transaction_gain_code=investment["D4"]["text"],
+                transaction_partner=investment["D5"]["text"],
+            )
+
+        for agreement in extracted_data["sections"]["Agreements"]["rows"]:
+            Agreements.objects.create(
+                financial_disclosure=disclosure,
+                redacted=True
+                in [v["is_redacted"] for _, v in agreement.items()],
+                date=agreement["Date"]["text"],
+                parties_and_terms=agreement["Parties and Terms"]["text"],
+            )
+
+        for debt in extracted_data["sections"]["Liabilities"]["rows"]:
+            Debt.objects.create(
+                financial_disclosure=disclosure,
+                redacted=True in [v["is_redacted"] for _, v in debt.items()],
+                creditor_name=debt["Creditor"]["text"],
+                description=debt["Description"]["text"],
+                value_code=debt["Value Code"]["text"],
+            )
+
+        for position in extracted_data["sections"]["Positions"]["rows"]:
+            Positions.objects.create(
+                financial_disclosure=disclosure,
+                redacted=True
+                in [v["is_redacted"] for _, v in position.items()],
+                position=position["Position"]["text"].replace("LL. ", ""),
+                organization_name=position["Name of Organization"]["text"],
+            )
+
+        for gift in extracted_data["sections"]["Gifts"]["rows"]:
+            Gift.objects.create(
+                financial_disclosure=disclosure,
+                source=gift["Source"]["text"],
+                description=gift["Description"]["text"],
+                value_code=gift["Value"]["text"],
+                redacted=True in [v["is_redacted"] for _, v in gift.items()],
+            )
+
+        for reimbursement in extracted_data["sections"]["Reimbursements"][
+            "rows"
+        ]:
+            if 5 != len(reimbursement.items()):
+                # Just in case - probably not needed
+                continue
```

Comment:
What's going on here? Need some detail.

Reply:
This can just be removed.  It was a bug that was cleaned up in the disclosure-extractor